src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaCloudformation.tsx

```diff
@@ -144,6 +145,16 @@ export default class AwsLambdaCloudformation extends React.Component<Props, Stat
     return !!region && testAccountNumber(accountNumber || '') && !!awsExternalId;
   }
 
+  trackOpenCloudFormation = () => {
+    //TODO: add org to trackIntegrationEvent call
```

Comment:
We don't actually have the org here so we'd have to pass it from the backend. For now, we don't need it.